libs/contact/integration-tests/hubspot.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { DaffContactHubspotService } from '../src/driver/hubspot/contact.service';
+import { DaffContactHubSpotDriverModule } from '../src/driver/hubspot/hubspot-driver.module';
+import { of } from 'rxjs';
+
+describe('DaffContactHubspotDriver', () => {
+  let contactService;
+  let httpMock: HttpTestingController;
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientTestingModule,
+        DaffContactHubSpotDriverModule.forRoot({ portalId: '123123', guid: '123123' })
+      ],
+      providers: [
+        DaffContactHubspotService
+      ]
+    });
+    httpMock = TestBed.get(HttpTestingController);
+    contactService = TestBed.get<DaffContactHubspotService>(DaffContactHubspotService);
+  });
+  afterEach(() => {
+    httpMock.verify();
+  });
+  it('should be created', () => {
+    expect(contactService).toBeTruthy();
+  });
+  describe('when sending', () => {
+    it('should send a submission', () => {
+      const forumSubmission = { email: 'test@email.com' };
+      const mockReq = of(forumSubmission);
+      contactService.send(forumSubmission).subscribe();
+      const req = httpMock.expectOne(`${'https://api.hsforms.com/submissions/v3/integration/submit/123123/123123'}`);
+      expect(req.request.body).toEqual(
+        {
+          fields: [Object({ 'name': 'email', 'value': 'test@email.com' })],
+          context: Object({ hutk: null, pageUri: null, pageName: '' })
+        }
+      );
+      req.flush(mockReq);
+      httpMock.verify();
```

Comment:
This duplicate `httpMock.verify` isn't necessary.

Reply:
All changes should be addressed in ac333252aa6378b2caf4f77f678fb62e65d91f08